test: smoke test so no need to click through

azul opened the pull request.

azul committed:
test: smoke test so no need to click through

hpk42 commented on ui.js:
```diff
@@ -1,42 +1,39 @@
+        'description': document.getElementById("description"),
+        'explanation': document.getElementById("explanation"),
+        'settings': document.getElementById("autocrypt-settings")
+    };
```
why not iterate over the list of Ids and put them to into the "dom" object?

azul replied:
Sounds good in general. Seems a bit unrelated to the pr though. I just moved the code out of the setup function but it also seems like i don't need to do that anymore. So i might just revert the changes to this file entirely.

azul committed:
use async to deal with firfox bug

We need to wait a few miliseconds before clicking on a button
we just enabled. Even though it may only be milliseconds it still
requires that write our tests in an async manner.
